frontend/src/components/pages/connect/CreateConnector.tsx

```diff
@@ -599,12 +608,13 @@ function getDataSource(validationResult: ConnectorValidationResult) {
 function ValidationDisplay({ validationResult }: { validationResult: ConnectorValidationResult }) {
     return (
         <Alert
-            style={{ marginTop: '2rem' }}
-            type="warning"
-            message={
-                <>
-                    <h3>Submitted configuration is invalid</h3>
-
+            status="warning"
+            variant="left-accent"
+            my={4}
+        >
+            <AlertDescription>
+                <Box>
+                    <Text as="h3" mb={4}>Submitted configuration is invalid</Text>
```

Comment:
@frankieshakes is there any typography text variant we should be using here, or is it fine as-is?

Reply:
The `as` is fine here if it _must_ be an H3 for semantic reasons. Otherwise, the default `<Text>` (with no variant) should suffice.

Reply:
I think semantically `h3` makes sense here, but we should probably create some styleguide what to use where in the future for the whole UI